src/components/common/Icons.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'proptypes';
+import classNames from 'classnames';
+
+const Icons = ({
+  id,
+  label,
+  handleClick,
+  iconTag,
+  iconSize,
+  /*
+   *  Props below correspond with Bulma modifiers.
+  */
+  color,
+  size,
+}) => {
+  // Dynamically generates infoIcon className from props to comply with Bulma styling modifiers.
+  const infoClassName = classNames('icon', {
+    [`is-${size}`]: size, //for small, meduim, large, span tag
+    [`has-text-${color}`]: color,
+    [`fas-fa${iconSize}`]:iconSize //for fa-lg, fa-2x, fa-3x, icon tag
+  });
```

Comment:
add `icon` to the classNames function so we can just add the name of the icon without the `fa-`

Reply:
Completed.